tests/fracs/test_fracture_utils.py

```diff
@@ -31,47 +27,47 @@ def nrm(x, y):
     return is_true
 
 
-class TestFractureLength(unittest.TestCase):
+class TestFractureLength:
```

Comment:
Why is this a class? I don't see the purpose.

Reply:
Agreed, changing it